workflows/cloudify_system_workflows/plugins.py

```diff
@@ -32,3 +35,43 @@ def _operate_on_plugin(ctx, plugin, action):
         'cloudify_agent.operations.{0}_plugins'.format(action),
         kwargs={'plugins': [plugin]}))
     return graph.execute()
+
+
+@workflow(system_wide=True)
+def update(ctx, update_id, temp_blueprint_id, deployments_to_update, **_):
+    """Execute deployment update for all the given deployments_to_update.
+
+    :param update_id: plugins update ID.
+    :param temp_blueprint_id: temporary blueprint ID that should be used for
+    this workflow only.
+    :param deployments_to_update: deployments to perform the update on, using
+    the temp blueprint ID provided.
+    """
+    err_msg_format = 'Deployment update has failed with execution ID: {}.'
+    client = get_rest_client()
+    for dep in deployments_to_update:
+        ctx.send_event('Executing deployment update for deployment '
+                       '{}...'.format(dep))
+        execution_id = client.deployment_updates \
+            .update_with_existing_blueprint(deployment_id=dep,
+                                            blueprint_id=temp_blueprint_id,
+                                            skip_install=True,
+                                            skip_uninstall=True,
+                                            skip_reinstall=True) \
+            .execution_id
+
+        wait_for(client.executions.get,
+                 execution_id,
+                 'status',
+                 lambda x: x in ExecutionState.END_STATES,
+                 RuntimeError,
+                 err_msg_format.format(execution_id))
+        execution_status = client.executions.get(execution_id).status
+        if execution_status in (ExecutionState.FAILED,
+                                ExecutionState.CANCELLED):
+            raise RuntimeError("Deployment update of deployment {0} with "
+                               "execution ID {1} failed, stopped this "
+                               "plugins update.".format(dep, execution_id))
```

Comment:
maybe add the plugin update id here, so the user could search for it

Reply:
You're absolutely right. Added.